book.adoc

```diff
@@ -26,6 +49,10 @@ Antoine Sabot-Durand <antoine@sabot-durand.net>
 === Getting and installing Tools used in this book
 === CDI website and resources
 === Choosing your implementation
+=== Using CDI with Java SE
+* Bootstraping using Weld / OWB
+=== Using CDI with Java EE
+* Arquillian
```

Comment:
You're right we should introudce Arquillian if we go for a Java EE app, but perhaps it should be introduced in "Getting and installing tools" or in a part just after.